backend/python/app/services/implementations/story_service.py

```diff
@@ -477,8 +485,8 @@ def update_story_translation_content_status(
 
     def approve_all_story_translation_content(self, story_translation_id):
```

Comment:
I wouldnt expect this mutation to be called in a soft-deleted story translation so I dont think this should use `StoryTranslationAll,` what do you think?